opentech/apply/workflow.py

```diff
@@ -5,8 +5,18 @@
 from django.forms import Form
 from django.utils.text import slugify
 
+"""
+This file defines classes which allow you to compose workflows based on the following strucutre:
+
+Workflow -> Stage -> Phase -> Action
```

Comment:
Having read through the comments, it makes a lot of sense to start the definitions from Action and work upwards.

Stage/Phase/Action is used with mixed case in docblocks. Let's keep it consistent.